webpack/localization-plugin/README.md

```diff
@@ -0,0 +1,181 @@
+By default, every `.resx` and `.loc.json` file import is intercepted by this plugin, and an error occurs
+if translations aren't provided for an intercepted file. To avoid that error, list files that should be ignored
+by this plugin in this property. Files should be specified as either absolute paths or paths relative
+to the Webpack compilation context.
+
+### `localizationStats = { }`
+
+#### `localizationStats.dropPath = '...'`
+
+This option is used to designate a path at which a JSON file describing the localized assets produced should be
+written. If this property is omitted, the stats file won't be written.
+
+The file has the following format:
+
+```JSON
+{
+  "entrypoints": {
+    "<BUNDLE NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    },
+    "<BUNDLE NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    }
+  },
+  "namedChunkGroups": {
+    "<CHUNK NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    },
+    "<CHUNK NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    }
+  }
+}
+
+```
+
+#### `localizationStats.callback = (stats) => { ... }`
+
+This option is used to specify a callback to be called with the stats data that would be dropped at
+[`localizationStats.dropPath`](#localizationStats.DropPath--) after compilation completes.
+
+### `typingsOptions = { }`
+
+This option is used to specify how and if TypeScript typings should be generated for loc files.
+
+It takes two options:
+
+#### `typingsOptions.generatedTsFolder = '...'`
+
+This property specifies the folder in which `.d.ts` files for loc files should be dropped. It is recommended
+that this be a folder parallel to the source folder, specified in addition to the source folder in the
+[`rootDirs` `tsconfig.json` option](https://www.typescriptlang.org/docs/handbook/compiler-options.html).
+**The folder specified by this option is emptied when compilation is started.**
+
+This property is required if `typingsOptions` is set.
+
+#### `typingsOptions.sourceRoot = '...'`
+
+This optional property overrides the compiler context for discovery of localization files for which
+typings should be generated.
+
+### `typingsOptions.exportAsDefault = true | false`
+
+If this option is set to `true`, loc modules typings will be exported wrapped in a `default` property. This
+allows strings to be imported by using the `import strings from './strings.loc.json';` syntax instead of
+the `import { string1 } from './strings.loc.json';` or the `import * as strings from './strings.loc.json';`
+syntax.
+
+# License
+
+MIT (http://www.opensource.org/licenses/mit-license.php)
```

Comment:
This markdown link isn't formatted correctly.  Also it's linking to a URL with a boilerplate attribution license, whereas the actual license is in the local LICENSE file.  Given that the license is already obvious from that file and the `license` field in package.json, I'd suggest to delete this section entirely.  That's consistent with our other projects.